pkg/cmd/attestation/verify/policy.go

```diff
@@ -42,17 +42,10 @@ func buildSANMatcher(opts *Options) (verify.SubjectAlternativeNameMatcher, error
 	return verify.SubjectAlternativeNameMatcher{}, nil
 }
 
-func buildCertExtensions(opts *Options, runnerEnv string) certificate.Extensions {
-	extensions := certificate.Extensions{
-		SourceRepositoryOwnerURI: fmt.Sprintf("https://github.com/%s", opts.Owner),
-		RunnerEnvironment:        runnerEnv,
+func buildCertExtensions(runnerEnv string) certificate.Extensions {
```

Comment:
doesn't need to be its own function anymore, can just pass in the variable

Reply:
updatd